templates/prusa-mk3s-printer.template.cfg

```diff
@@ -212,10 +201,14 @@ recover_velocity: 50.
 position_max: 214
 
 [probe]
-z_offset = 2.0
+z_offset = -0.0
```

Comment:
remove  `-`  so it's just `0.0`

Reply:
Done, I use "-" in order to indicate that should be a negative value